src/metrics/migrations/0006_auto_20190627_1412.py

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+import re
+
+from django.db import migrations
+
+from metrics.logic import get_iso_country_code
+
+
+def process_accesses(apps, schema_editor):
+    ArticleAccess = apps.get_model('metrics', 'ArticleAccess')
+    Country = apps.get_model('core', 'Country')
+
+    accesses = ArticleAccess.objects.all()
+
+    for access in accesses:
+        print(access.pk)
+        check = re.match(
+            r"^\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3}$",
+            access.identifier,
+        )
+
+        if check:
+            print('ip match')
+
+            try:
+                code = get_iso_country_code(access.identifier)
+                country = Country.objects.get(code=code)
+            except Country.DoesNotExist:
+                country = None
+
+            access.country = country
+            access.save()
+
+            print(access.country)
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('metrics', '0005_articleaccess_country'),
```

Comment:
Should have a dependancy on `core` as well since it is using `core.Country`

Reply:
Core dependency is in 0005.

src/metrics/migrations/0005_articleaccess_country.py